ansibullbot/utils/extractors.py

```diff
@@ -570,3 +577,50 @@ def get_module_metadata(self):
             pass
 
         return meta
+
+    def get_deprecation_info(self):
+        dmeta = {
+            'deprecated': False,
+            'alternatives': None,
+            'deprecated_in': None,
+            'removed_in': None,
+            'why': None
+        }
+
+        documentation = ''
+        inphase = False
+
+        lines = self.filedata.split('\n')
+        for line in lines:
+            if 'DOCUMENTATION' in line:
+                inphase = True
+                continue
+            if inphase and (line.strip().endswith("'''") or line.strip().endswith('"""')):
+                #phase = None
+                break
+            if inphase:
+                documentation += line + '\n'
+
+        if not documentation:
+            logging.debug('no documentation found in {}'.format(self.filepath))
+            return []
+
+        if 'deprecat' not in documentation:
+            return dmeta
+
+        docs_dict = yaml.load(documentation)
+        if 'deprecated' not in docs_dict:
+            return dmeta
+
+        dmeta['deprecated'] = True
+        for k,v in docs_dict['deprecated'].items():
+            dmeta[k] = v
+            if k == 'alternative':
+                dmeta['alternatives'] = []
+                alts = v.split()
+                alts = [x for x in alts if x.startswith('M(')]
+                alts = [x.replace('M(', '') for x in alts]
```

Comment:
Need to strip out `C(...)` and `U(...)`

`_nxos_mtu.py` has:
`alternative: Use M(nxos_system)'s C(system_mtu) option. To specify an interfaces MTU use M(nxos_interface).`